libs/sr/formula/src/data/lightCone/sheets/PastSelfInMirror.ts

```diff
@@ -20,28 +16,17 @@ const dm = mappedStats.lightCone[key]
 
 const { superimpose } = own.lightCone
 
-// TODO: Add conditionals
-const { boolConditional } = allBoolConditionals(key)
-const { listConditional } = allListConditionals(key, ['val1', 'val2'])
-const { numConditional } = allNumConditionals(key, true, 0, 2)
+const { useUltimate } = allBoolConditionals(key)
 
 const sheet = register(
   key,
   // Handles base stats and passive buffs
   entriesForLightCone(data_gen),
 
-  // TODO: Add formulas/buffs
   // Conditional buffs
   registerBuff(
     'cond_dmg_',
-    ownBuff.premod.dmg_.add(
-      boolConditional.ifOn(subscript(superimpose, dm.cond_dmg_))
-    )
-  ),
-  registerBuff(
-    'team_dmg_',
-    teamBuff.premod.dmg_.add(listConditional.map({ val1: 1, val2: 2 }))
-  ),
-  registerBuff('enemy_defIgn_', enemyDebuff.common.defIgn_.add(numConditional))
+    teamBuff.premod.dmg_.add(useUltimate.ifOn(subscript(superimpose, dm.dmg_)))
```

Comment:
@nguyentvan7  Should this be conditional behind a lightcone accumulator? like how we do a set counter for artifacts?